File: CS/FillEmptySpaceGridColumn/FillGridColumnCollection.cs
using DevExpress.XtraGrid.Columns;
using DevExpress.XtraGrid.Views.Base;

namespace FillEmptySpaceGridColumn
{
	public class FillGridColumnCollection : GridColumnCollection
	{
		public FillGridColumnCollection(ColumnView view)
			: base(view)
		{
		}

		protected override GridColumn CreateColumn()
		{
			return new FillGridColumn();
		}
	}
}

File: VB/FillEmptySpaceGridColumn/FillGridColumnCollection.vb
Imports DevExpress.XtraGrid.Columns
Imports DevExpress.XtraGrid.Views.Base

Namespace FillEmptySpaceGridColumn
    Public Class FillGridColumnCollection
        Inherits GridColumnCollection

        Public Sub New(ByVal view As ColumnView)
            MyBase.New(view)
        End Sub

        Protected Overrides Function CreateColumn() As GridColumn
            Return New FillGridColumn()
        End Function
    End Class
End Namespace
